File: lane_stat.py
#!/usr/bin/python
# -*- coding:utf-8 -*-
import sys
if len(sys.argv) < 2:
	print "python",sys.argv[0],"<info.csv>"
	exit(1)
dic = {}
f = open(sys.argv[1],'r')
f.readline() # skip the first line
for i in f:
	i = i.strip().split(",")
	key,value = i[0],i[3]
	lst = dic.setdefault(key,[])
	lst.append(float(value))
for k in sorted(dic):
	s = sum(dic[k])
	print k,s

File: lane_stat.pl
#!/usr/bin/perl -w
use strict;
die "perl $0 <info.csv>\n" unless @ARGV == 1;
my $info = $ARGV[0];
open IN,$info;
<IN>;
my %hash;
while(<IN>){
  chomp;
  my @arr = split/,/;
  my $key = $arr[0];
  my $value = $arr[3];
  $hash{$key} += $value;
}
close IN;
foreach my $k( sort keys %hash){
  print $k,"\t",$hash{$k},"\n";
}
